services/kommander/0.6.0/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+  - dynamic-helmreleases/mtls-proxy/defaults/cm.yaml
```

Comment:
(discussed in standup but commenting for posterity) -- this means the default cm only gets deployed once into the kommander ns -- but the mtls-proxy HR is deployed into workspace namespaces, so I believe the cm's would need to exist in the same namespace

Reply:
yup thanks, i'll need to see if its possible to deploy once per WS with this current structure 🤔

Reply:
i think i'll have to update the controller to create a Kustomization pointing to just the defaults CM directory

Reply:
Ah man that is annoying, thanks for your diligence @gracedo